samples/javascript_nodejs/14.nlp-with-dispatch/homeAutomation.js

```diff
@@ -47,61 +47,61 @@ class homeAutomation {
         });
     }
     /**
-     * 
-     * @param {Object} context context object
+     *
+     * @param {TurnContext} context context object
      */
-    async onTurn(context) {
+    async onTurn(turnContext) {
```

Comment:
do one of two things, either change thus parameter from turnContext to context, or change line 51 change context to turnContext in the param comment

Reply:
in other places you're using 'context' as the param name

Reply:
Updated all to be turnContext.